indico/web/client/js/react/components/WTFSearchDropdown.module.scss

```
@@ -0,0 +1,15 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+:global(.ui.multiple.search.dropdown) :global(input[type='text'].search) {
+  margin: 0em 0.64285714em;
```

```suggestion
  margin: 0 0.64285714em;
```

`0` usually needs no unit